documentation/modules/uninstalling-mtv-cli.adoc

```diff
@@ -6,31 +6,38 @@
 [id="uninstalling-mtv-cli_{context}"]
 = Uninstalling {project-short} from the command line interface
 
-You can uninstall {project-first} from the command line interface (CLI) by deleting the +{namespace}+ project and the `forklift.konveyor.io` custom resource definitions (CRDs).
+You can uninstall {project-first} from the command line interface (CLI) by deleting the `forklift` controller, the {project-short} Operator, and the plugin console custom resource (CR).
 
 .Prerequisites
 
 * You must be logged in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Delete the project:
+. Delete the `forklift` controller by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} delete project {namespace}
+oc delete ForkliftController --all -n openshift-mtv
 ----
 
-. Delete the CRDs:
+. Delete the {project-short} Operator by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} get crd -o name | grep 'forklift' | xargs {oc} delete
+oc delete Subscription mtv-operator -n openshift-mtv
 ----
 
-. Delete the OAuthClient:
+. Delete the plugin console CR by running the following command:
```

Comment:
```suggestion
. Delete the plugin console CR:
```

Reply:
@apurvabhide17 For OpenShift, we need to include "by running the following command" or a similar phrase for each and every command. So, in the name of consistency between repos, I'm going to keep my phrasing.